src/pages/compose/index.tsx

```diff
@@ -145,143 +141,145 @@ const Compose = () => {
-                  marginLeft: "6rem",
-                  padding:"0.5rem",
-                  opacity: projectId ? 1 : 0.5
-                }}
-                
-                onClick={projectId ? () => handleIconClick("HddFilled") : () => {} }
-                alt=""
-              />{" "}
-              <br />
-
-              <Image
-                preview={false}
-                src="/groups-icon.png"
-                style={{
-                  ...getIconStyle("ContainerFilled"),
-                  width: "3.5rem",
-                  height: "3.5rem",
-                  marginLeft: "6rem",
-                 padding:"0.5rem",
-                  opacity: projectId ? 1 : 0.5
-                }}
-                alt=""
-                onClick={projectId ? () => handleIconClick("ContainerFilled"): () => {}}
+      <Space direction="vertical" className={styles.space} size={[0, 48]}>
+        <Layout className={styles.layout}>
+          <Header />
+          <Content style={{ marginTop: "1rem" }}>
+            <Row>
+              <TextAreaComponent
+                onNameChange={setName}
+                nameValue={name}
+                onDescriptionChange={setDescription}
+                descriptionValue={description}
+                nameError={nameError}
+                descriptionError={descriptionError}
+                className={nameError ? "textAreaError" : ""}
               />
-              <br />
-
-              <Image
-                preview={false}
-                src="/compose-pipeline.png"
-                style={{
-                  ...getIconStyle("ComposePipeline"),
-                  width: "3.5rem",
-                  height: "3.5rem",
-                  marginLeft: "6rem",
-               
-                  padding:"0.3rem",
-                  opacity: projectId ? 1 : 0.5
-                }}
-                alt=""
-                onClick={projectId ? () => handleIconClick("ComposePipeline"): () => {}}
+              <Col
+                span={1}
+                className={styles.textAreaBorder}
+                style={{ borderRight: "1px solid #ccc" }}
+              ></Col>
+              <ButtonComponent
+                saveModalVisible={saveModalVisible}
+                handleSaveModalOk={handleSaveProjectInfo}
+                handleSaveModalCancel={handleSaveModalCancel}
+                handleSaveClick={handleSaveClick}
+                handleDeleteClick={handleDeleteClick}
+                handleDeleteModalOk={handleDeleteModalOk}
+                handleDeleteModalCancel={handleDeleteModalCancel}
+                deleteModalVisible={deleteModalVisible}
+                saveBoxMessage={
+                  nameError
+                    ? "name can not be empty"
+                    : descriptionError
+                    ? "description can not be empty"
+                    : nameError && descriptionError
+                    ? "name and description can not be empty"
```

Comment:
The hardcoded values can be put in constant files

Reply:
Done